v1.1/admin_ui_replication_dashboard.md

```diff
@@ -0,0 +1,62 @@
+- **Range Replica:** CockroachDB replicates each range (3 times by default) and stores each replica on a different node.
+- **Range Lease:** For each range, one of the replicas holds the "range lease". This replica, referred to as the "leaseholder", is the one that receives and coordinates all read and write requests for the range.
+- **Under-replicated Ranges:** When a cluster is first initialized, the few default starting ranges will only have a single replica, but as soon as other nodes are available, they will replicate to them until they've reached their desired replication factor, the default being 3. If a range does not have enough replicas, the range is said to be "under-replicated".
+- **Unavailable Ranges:** If a store has not been heard from in some time, the default setting being 5 minutes, the cluster will consider this store to be dead. When this happens, all ranges that have replicas on that store are determined to be "unavailable" and removed.
+
+## Replication Dashboard
+
+The Replication dashboard displays the following time-series graphs:
+
+### Ranges
+<img src="{{ 'images/admin_ui_ranges.png' | relative_url }}" alt="CockroachDB Admin UI Replicas per Store" style="border:1px solid #eee;max-width:100%" />
+The graph shows the number of ranges across all nodes in the cluster.
+On hovering over the graph, the values for the following metrics are displayed:
+
+Metric | Description
+--------|----
+Ranges | The number of ranges across all nodes in the cluster.
+Leaders | The number of ranges with leaders across all nodes in the cluster. If the number does not match number of ranges for a long time, troubleshoot your cluster.
+Lease Holders | The number of ranges that have leases across all nodes in the cluster.
+Leaders w/o Leases | The number of Raft leaders without leases across all nodes in the cluster. If the number if non-zero for a long time, troubleshoot your cluster. 
+Unavailable | The number of unavailable ranges across all nodes in the cluster. If the number if non-zero for a long time, troubleshoot your cluster. 
+Under-replicated | The number of under-replicated ranges across all nodes in the cluster.
+
+### Replicas Per Store
+<img src="{{ 'images/admin_ui_replicas_per_store.png' | relative_url }}" alt="CockroachDB Admin UI Replicas per Store" style="border:1px solid #eee;max-width:100%" />
+The graph displays the number of range replicas on each store. 
+
+By default, the cluster-wide replication zone is set to replicate data to any three nodes in your cluster, with ranges in each replica splitting once they get larger than 64 MB. The replicas are balanced evenly across the nodes. You can [Configure replication zones](configure-replication-zones.html) to set the number and location of replicas. You can monitor the changes in configuration using the Admin UI, as described in [Fault tolerance and recovery](demo-fault-tolerance-and-recovery.html).
```

Comment:
I would remove the two sentences: "By default, the cluster-wide replication zone is set to replicate data to any three nodes in your cluster, with ranges in each replica splitting once they get larger than 64 MB. The replicas are balanced evenly across the nodes." Otherwise they are bound to become stale when we change the defaults. The sentence immediately afterwards will point the reader in the right direction anyway.